src/main/java/com/elytradev/concrete/inventory/gui/ConcreteContainer.java

```diff
@@ -225,7 +224,7 @@ public ItemStack transferToInventory(ItemStack stack, IInventory inventory) {
 	public boolean canStackTogether(ItemStack src, ItemStack dest) {
 		if (src.isEmpty() || dest.isEmpty()) return false; //Don't stack using itemstack counts if one or the other is empty.
 
-        boolean compoundComparison = false;
+        boolean compoundComparison;
```

Comment:
While booleans default to false, I object to intentionally relying on values for uninitialized variables.